docs/csharp/programming-guide/classes-and-structs/ref-returns.md

```diff
@@ -3,40 +3,35 @@ title: "Ref return values and ref locals (C# Guide)"
 description: "Learn how to define and use ref return and ref local values"
 author: "rpetrusha"
 ms.author: "ronpet"
-ms.date: "05/30/2017"
+ms.date: "01/23/2017"
 ms.topic: "article"
 ms.prod: ".net"
 ms.technology: "devlang-csharp"
 ms.devlang: "csharp"
-ms.assetid: "18cf7a4b-29f0-4b14-85b8-80af754aabd8"
 ---
 # Ref returns and ref locals
 
-Starting with C# 7, C# supports reference return values (ref returns). A reference return value allows a method to return a reference to an object, rather than a value, back to a caller. The caller can then choose to treat the returned object returned as if it were returned by value or by reference. A value returned by reference that the caller handles as a reference rather than a value is a ref local.
+Starting with C# 7, C# supports reference return values (ref returns). A reference return value allows a method to return a reference to a variable, rather than a value, back to a caller. The caller can then choose to treat the returned variable as if it were returned by value or by reference. The caller can create a new variable that is itself a reference to the returned value, called a ref local.
 
 ## What is a reference return value?
 
-Most developers are familiar with passing an argument to a called method *by reference*. A called method's argument list includes a value passed by reference, and any changes made to its value by the called method are returned to the caller. A *reference return value* is the opposite:
+Most developers are familiar with passing an argument to a called method *by reference*. A called method's argument list includes a variable passed by reference, and any changes made to its value by the called method are observed by the caller. A *reference return value* means that a method returns a *reference* (or an alias) to some variable whose scope includes the method and whose lifetime must extend beyond the return of the method.
 
```

Comment:
It's somewhat unclear what the following list is supposed to represent. There's no transition to it from this paragraph.